docs/development/update-project-dependency.md

```diff
@@ -5,29 +5,46 @@ EKS-D matches the versions used in the cloud EKS service and generally dependenc
 during the life cycle of a specific Kubernetes release life cycle.  To account for security related fixes upstream
 we occasionally need to update specific dependencies for some of the projects in this repo.
 
-## Typical workflow
-
-1. Based on our automated Dependeabot alerts or Inspector V2 scans, identify dependencies in need of updating
-along with the fix version. **Note** Be sure to include the `RELEASE_BRANCH=<>` along with each make call
-if updating a released branch project. It may be easier to just `export RELEASE_BRANCH=<>` before starting this process.
-1. In the specific project folder:
+## Typical workflo
+1. **Identify dependencies in need of updating**: Be sure to include the `RELEASE_BRANCH=<>` along with each make call
+if updating a released branch project.
+    * Based on our automated Dependeabot alerts or Inspector V2 scans
```

Comment:
I know the original doc included references to inspector V2, but it this something we're not trying to make super public? I'm asking because I genuinely don't know if we are or not

Reply:
Good call out